CHANGELOG.md

```diff
@@ -6,14 +6,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Buy button support to add multiple sku items to the cart at the same time.
```

Comment:
Put the component name that you changed into backtick to be like this `BuyButton`.